### PR TITLE
[RHBOP] Exclude kubernetes module from productized profile

### PR DESCRIPTION
**Pull request by lampajr**
### JIRA
- https://issues.redhat.com/browse/BXMSPROD-1826

### Referenced pull requests
- https://github.com/kiegroup/optaplanner/pull/2224

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

- for <b>pull request checks</b>  
  Please add comment: <b>Jenkins retest this</b>

- for a <b>specific pull request check</b>  
  please add comment: <b>Jenkins (re)run [optaplanner-quickstarts] tests</b>

- for a <b>full downstream build</b>  
  please add the label `run_fdb`

- for <b>quarkus branch checks</b>  
  Run checks against Quarkus current used branch  
  Please add comment: <b>Jenkins run quarkus-branch</b>

- for a <b>quarkus branch specific check</b>  
  Run checks against Quarkus current used branch  
  Please add comment: <b>Jenkins (re)run [optaplanner-quickstarts] quarkus-branch</b>

- for <b>quarkus main checks</b>  
  Run checks against Quarkus main branch  
  Please add comment: <b>Jenkins run quarkus-main</b>

- for a <b>specific quarkus main check</b>  
  Run checks against Quarkus main branch  
  Please add comment: <b>Jenkins (re)run [optaplanner-quickstarts] quarkus-branch</b>

- for <b>native checks</b>  
  Run native checks  
  Please add comment: <b>Jenkins run native</b>

- for a <b>specific native check</b>  
  Run native checks 
  Please add comment: <b>Jenkins (re)run [optaplanner-quickstarts] native</b>

- for <b>mandrel checks</b>  
  Run native checks against Mandrel image
  Please add comment: <b>Jenkins run mandrel</b>

- for a <b>specific mandrel check</b>  
  Run native checks against Mandrel image  
  Please add comment: <b>Jenkins (re)run [optaplanner-quickstarts] mandrel</b>
</details>

### CODE REVIEW

**Comment by mareknovotny** [comment]
i thought we will use the assembly project to include/exclude module sources at the end
We don't need for that productized profile

**Comment by lampajr** [comment]
> i thought we will use the assembly project to include/exclude module sources at the end
> We don't need for that productized profile 

I agree on using the assembly to exclude those modules, but if on QE side they will use the pom.xml don't we need to explicitly remove that module from the pom itself? (otherwise it will look for it but it won't find it). Maybe I'm missing something :)

**Comment by mareknovotny** [comment]
qe should test the sources we deliver from build nightly or from candidate area. So pom.xml in github repo stays the same without a need for changes IMO.

**Comment by lampajr** [comment]
@mareknovotny Ok make sense, therefore I should be able to manage everything through the rhbop-optaplanner.
I will close this PR.

**Comment by lampajr** [comment]
As agreed adding `productized` profile seems to be the best solution to allow `technology/kuebernetes` module to be excluded during the build. Therefore I'm reopening this PR.

File: pom.xml [hunk 1]
```diff
@@ -33,6 +32,18 @@
...
   <profiles>
+    <profile>
+      <id>default</id>
+      <activation>
+        <property>
+          <name>!productized</name>
+        </property>
+      </activation>
+      <modules>
+        <module>technology/kubernetes</module>
```
Comment (Ginxo) [inline pom.xml:43]: this should be present on `fullProfile` profile right?

**Review by Ginxo — CHANGES_REQUESTED** [review]
please review my comments

File: pom.xml (hunk 1 above)
Comment (lampajr) [reply pom.xml:43]: yeah, I guess so

**Review by Ginxo — APPROVED** [review]
Forget about my previous comment... LGTM

**Comment by Ginxo** [comment]
@ge0ffrey can you please provide feedback? Thanks

File: pom.xml (hunk 1 above)
Comment (mareknovotny) [reply pom.xml:43]: hmm, not sure if we won't run fullProfile too? Then it would be not wanted, right? @Ginxo

**Review by ge0ffrey — APPROVED** [review]

**Review by ge0ffrey — CHANGES_REQUESTED** [review]
This should be against the development branch. The stable branch must not be changed by PR's.

**Comment by mareknovotny** [comment]
i can't change the target here as it will produce too many conflicts,  @lampajr please take your commits and base them on development branch for a new PR

**Comment by kie-ci3** [comment]
**(tests) - optaplanner-quickstarts job** `#82` was: **FAILURE**
Possible explanation: Pipeline failure or project build failure

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/optaplanner/job/main/job/pullrequest/job/optaplanner-quickstarts.tests.optaplanner-quickstarts/82/display/redirect
See console log:
<details>
<summary><b>Console Logs</b></summary>
Fetching changes from the remote Git repository<br/> > git config remote.origin.url https://github.com/kiegroup/jenkins-pipeline-shared-libraries # timeout=10<br/>Fetching without tags<br/>Fetching upstream changes from https://github.com/kiegroup/jenkins-pipeline-shared-libraries<br/> > git --version # timeout=10<br/> > git --version # 'git version 2.31.1'<br/>using GIT_SSH to set credentials SSH: kie-ci service account with access to jboss-integration private github space<br/>[INFO] Currently running in a labeled security context<br/> > /usr/bin/chcon --type=ssh_home_t /var/jenkins_home/jobs/KIE/jobs/optaplanner/jobs/main/jobs/pullrequest/jobs/optaplanner-quickstarts.tests.optaplanner-quickstarts/workspace@libs/d03f8b53ef89e88ec2b15428a250a4b1fb7910bfcdd3e1c33e4e95c3adb758bb@tmp/jenkins-gitclient-ssh9178842326168817628.key<br/>Verifying host key using known hosts file, will automatically accept unseen keys<br/> > git fetch --no-tags --force --progress -- https://github.com/kiegroup/jenkins-pipeline-shared-libraries +refs/heads/*:refs/remotes/origin/* # timeout=10<br/>Checking out Revision c1856f92a564bf4129d0d912762042e28b59041f (main)<br/> > git config core.sparsecheckout # timeout=10<br/> > git checkout -f c1856f92a564bf4129d0d912762042e28b59041f # timeout=10<br/>Commit message: "added method for open a PR as draft (#219)"<br/> > git rev-list --no-walk c1856f92a564bf4129d0d912762042e28b59041f # timeout=10<br/>[Pipeline] Start of Pipeline<br/>[Pipeline] node<br/>Still waiting to schedule task<br/>Waiting for next available executor on ‘kie-rhel7&&kie-mem16g&&!built-in’<br/>Running on rhos-01-kie-psi-rhel7-xlarge-167 in /home/jenkins/workspace/KIE/optaplanner/main/pullrequest/optaplanner-quickstarts.tests.optaplanner-quickstarts<br/>[Pipeline] {<br/>[Pipeline] stage<br/>[Pipeline] { (Declarative: Checkout SCM)<br/>[Pipeline] checkout<br/>The recommended git tool is: NONE<br/>using credential kie-ci<br/>Cloning the remote Git repository<br/>Cloning repository https://github.com/kiegroup/kogito-pipelines.git<br/> > git init /home/jenkins/workspace/KIE/optaplanner/main/pullrequest/optaplanner-quickstarts.tests.optaplanner-quickstarts # timeout=10<br/>Fetching upstream changes from https://github.com/kiegroup/kogito-pipelines.git<br/> > git --version # timeout=10<br/> > git --version # 'git version 2.37.1'<br/>using GIT_ASKPASS to set credentials PSSWD: kie-ci service account with access to jboss-integration private github space<br/> > git fetch --tags --force --progress -- https://github.com/kiegroup/kogito-pipelines.git +refs/heads/*:refs/remotes/origin/* # timeout=10<br/>Avoid second fetch<br/>Checking out Revision d5088cfc265a7b80bc94f213267e7f20c79ff0a1 (origin/main)<br/> > git config remote.origin.url https://github.com/kiegroup/kogito-pipelines.git # timeout=10<br/> > git config --add remote.origin.fetch +refs/heads/*:refs/remotes/origin/* # timeout=10<br/> > git rev-parse origin/main^{commit} # timeout=10<br/> > git config core.sparsecheckout # timeout=10<br/> > git checkout -f d5088cfc265a7b80bc94f213267e7f20c79ff0a1 # timeout=10<br/>Commit message: "upgrade build-chain to version 2.6.25 (#645)"<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] withEnv<br/>[Pipeline] {<br/>[Pipeline] withEnv<br/>[Pipeline] {<br/>[Pipeline] timestamps<br/>[Pipeline] {<br/>[Pipeline] timeout<br/>[2022-10-06T08:28:04.210Z] Timeout set to expire in 3 hr 0 min<br/>[Pipeline] {<br/>[Pipeline] stage<br/>[Pipeline] { (Declarative: Tool Install)<br/>[Pipeline] tool<br/>[2022-10-06T08:28:04.284Z] Installing NodeJS from /var/jenkins_home/caches/nodejs/LINUX/amd64/16.2.0.tar.gz to /home/jenkins/tools/jenkins.plugins.nodejs.tools.NodeJSInstallation/nodejs-16.2.0 on rhos-01-kie-psi-rhel7-xlarge-167<br/>[Pipeline] envVarsForTool<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] withEnv<br/>[Pipeline] {<br/>[Pipeline] stage<br/>[Pipeline] { (Initialize)<br/>[Pipeline] tool<br/>[Pipeline] envVarsForTool<br/>[Pipeline] withEnv<br/>[Pipeline] {<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>[2022-10-06T08:28:06.449Z] + printenv<br/>[Pipeline] archiveArtifacts<br/>[2022-10-06T08:28:06.518Z] Archiving artifacts<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] }<br/>[Pipeline] // withEnv<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (check space before build)<br/>[Pipeline] tool<br/>[Pipeline] envVarsForTool<br/>[Pipeline] withEnv<br/>[Pipeline] {<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] dir<br/>[2022-10-06T08:28:07.048Z] Running in /home/jenkins/workspace/KIE/optaplanner/main/pullrequest/optaplanner-quickstarts.tests.optaplanner-quickstarts<br/>[Pipeline] {<br/>[Pipeline] echo<br/>[2022-10-06T08:28:07.073Z] [INFO] space left on the machine<br/>[Pipeline] sh<br/>[2022-10-06T08:28:07.352Z] + df -h<br/>[2022-10-06T08:28:07.352Z] Filesystem      Size  Used Avail Use% Mounted on<br/>[2022-10-06T08:28:07.352Z] devtmpfs        7.8G     0  7.8G   0% /dev<br/>[2022-10-06T08:28:07.352Z] tmpfs           7.8G     0  7.8G   0% /dev/shm<br/>[2022-10-06T08:28:07.352Z] tmpfs           7.8G   17M  7.8G   1% /run<br/>[2022-10-06T08:28:07.352Z] tmpfs           7.8G     0  7.8G   0% /sys/fs/cgroup<br/>[2022-10-06T08:28:07.352Z] /dev/vda1        80G   20G   61G  24% /<br/>[2022-10-06T08:28:07.352Z] tmpfs           1.6G     0  1.6G   0% /run/user/1000<br/>[Pipeline] echo<br/>[2022-10-06T08:28:07.369Z] [INFO] space of /home/jenkins<br/>[Pipeline] sh<br/>[2022-10-06T08:28:07.649Z] + du -h -d1 /home/jenkins<br/>[2022-10-06T08:28:07.649Z] 4.0K	/home/jenkins/.ssh<br/>[2022-10-06T08:28:07.649Z] 0	/home/jenkins/go<br/>[2022-10-06T08:28:07.649Z] 8.0K	/home/jenkins/.vnc<br/>[2022-10-06T08:28:07.649Z] 8.0K	/home/jenkins/.m2<br/>[2022-10-06T08:28:07.649Z] 2.1G	/home/jenkins/git-repos<br/>[2022-10-06T08:28:07.649Z] 8.0K	/home/jenkins/.config<br/>[2022-10-06T08:28:07.649Z] 4.0K	/home/jenkins/.cekit<br/>[2022-10-06T08:28:07.649Z] 82M	/home/jenkins/virtenvs<br/>[2022-10-06T08:28:07.649Z] 32M	/home/jenkins/remoting<br/>[2022-10-06T08:28:07.649Z] 0	/home/jenkins/.cache<br/>[2022-10-06T08:28:07.649Z] 3.0M	/home/jenkins/workspace<br/>[2022-10-06T08:28:07.649Z] 0	/home/jenkins/.pki<br/>[2022-10-06T08:28:07.649Z] 100M	/home/jenkins/tools<br/>[2022-10-06T08:28:07.649Z] 0	/home/jenkins/caches<br/>[2022-10-06T08:28:07.649Z] 2.4G	/home/jenkins<br/>[Pipeline] echo<br/>[2022-10-06T08:28:07.665Z] [INFO] space of workspace<br/>[Pipeline] sh<br/>[2022-10-06T08:28:07.946Z] + du -h -d3 /home/jenkins/workspace<br/>[2022-10-06T08:28:07.946Z] 3.0M	/home/jenkins/workspace/KIE/optaplanner/main<br/>[2022-10-06T08:28:07.946Z] 3.0M	/home/jenkins/workspace/KIE/optaplanner<br/>[2022-10-06T08:28:07.946Z] 3.0M	/home/jenkins/workspace/KIE<br/>[2022-10-06T08:28:07.946Z] 3.0M	/home/jenkins/workspace<br/>[Pipeline] }<br/>[Pipeline] // dir<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] }<br/>[Pipeline] // withEnv<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Install build-chain tool)<br/>[Pipeline] tool<br/>[Pipeline] envVarsForTool<br/>[Pipeline] withEnv<br/>[Pipeline] {<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] echo<br/>[2022-10-06T08:28:08.206Z] [INFO] Getting build-chain version from composite action file<br/>[Pipeline] readFile<br/>[Pipeline] echo<br/>[2022-10-06T08:28:08.416Z] [INFO] build-chain version recovered '2.3.36'<br/>[Pipeline] sh<br/>[2022-10-06T08:28:08.705Z] + npm install -g @kie/build-chain-action@2.3.36 -registry=https://repository.engineering.redhat.com/nexus/repository/registry.npmjs.org/<br/>[2022-10-06T08:28:10.073Z] npm WARN ERESOLVE overriding peer dependency<br/>[2022-10-06T08:28:10.073Z] npm WARN Found: @octokit/core@2.5.4<br/>[2022-10-06T08:28:10.073Z] npm WARN node_modules/@kie/build-chain-action/node_modules/@octokit/core<br/>[2022-10-06T08:28:10.073Z] npm WARN   @octokit/core@"^2.4.3" from @octokit/rest@17.11.2<br/>[2022-10-06T08:28:10.073Z] npm WARN   node_modules/@kie/build-chain-action/node_modules/@octokit/rest<br/>[2022-10-06T08:28:10.073Z] npm WARN     @octokit/rest@"^17.6.0" from @kie/build-chain-action@2.3.36<br/>[2022-10-06T08:28:10.073Z] npm WARN     node_modules/@kie/build-chain-action<br/>[2022-10-06T08:28:10.073Z] npm WARN     1 more (@kie/build-chain-configuration-reader)<br/>[2022-10-06T08:28:10.073Z] npm WARN   1 more (@octokit/plugin-paginate-rest)<br/>[2022-10-06T08:28:10.073Z] npm WARN <br/>[2022-10-06T08:28:10.073Z] npm WARN Could not resolve dependency:<br/>[2022-10-06T08:28:10.073Z] npm WARN peer @octokit/core@">=3" from @octokit/plugin-request-log@1.0.4<br/>[2022-10-06T08:28:10.073Z] npm WARN node_modules/@kie/build-chain-action/node_modules/@octokit/plugin-request-log<br/>[2022-10-06T08:28:10.073Z] npm WARN   @octokit/plugin-request-log@"^1.0.0" from @octokit/rest@17.11.2<br/>[2022-10-06T08:28:10.073Z] npm WARN   node_modules/@kie/build-chain-action/node_modules/@octokit/rest<br/>[2022-10-06T08:28:13.342Z] <br/>[2022-10-06T08:28:13.342Z] added 111 packages, and audited 112 packages in 5s<br/>[2022-10-06T08:28:13.342Z] <br/>[2022-10-06T08:28:13.342Z] 5 packages are looking for funding<br/>[2022-10-06T08:28:13.342Z]   run `npm fund` for details<br/>[2022-10-06T08:28:13.342Z] <br/>[2022-10-06T08:28:13.342Z] found 0 vulnerabilities<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] }<br/>[Pipeline] // withEnv<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Build projects)<br/>[Pipeline] tool<br/>[Pipeline] envVarsForTool<br/>[Pipeline] tool<br/>[Pipeline] envVarsForTool<br/>[Pipeline] tool<br/>[Pipeline] envVarsForTool<br/>[Pipeline] withEnv<br/>[Pipeline] {<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] echo<br/>[2022-10-06T08:28:13.679Z] BUILD_MVN_OPTS_CURRENT = -Dfull<br/>[Pipeline] configFileProvider<br/>[2022-10-06T08:28:13.692Z] provisioning config files...<br/>[2022-10-06T08:28:13.701Z] copy managed file [kogito_pr_settings] to file:/home/jenkins/workspace/KIE/optaplanner/main/pullrequest/optaplanner-quickstarts.tests.optaplanner-quickstarts@tmp/config7090751057628717064tmp<br/>[Pipeline] {<br/>[Pipeline] withCredentials<br/>[2022-10-06T08:28:13.733Z] Masking supported pattern matches of $GITHUB_TOKEN<br/>[Pipeline] {<br/>[Pipeline] sh<br/>[2022-10-06T08:28:13.761Z] Warning: A secret was passed to "sh" using Groovy String interpolation, which is insecure.<br/>[2022-10-06T08:28:13.762Z] 		 Affected argument(s) used the following variable(s): [GITHUB_TOKEN]<br/>[2022-10-06T08:28:13.762Z] 		 See https://jenkins.io/redirect/groovy-string-interpolation for details.<br/>[2022-10-06T08:28:14.071Z] + build-chain-action -token=**** -df=https://raw.githubusercontent.com/kiegroup/kogito-pipelines/main/.ci/pull-request-config.yaml -folder=bc build pr -url=https://github.com/kiegroup/optaplanner-quickstarts/pull/421 -sp=kiegroup/optaplanner-quickstarts --skipParallelCheckout -cct '(^mvn .*)||$1 -s /home/jenkins/workspace/KIE/optaplanner/main/pullrequest/optaplanner-quickstarts.tests.optaplanner-quickstarts@tmp/config7090751057628717064tmp -Dmaven.wagon.http.ssl.insecure=true -Dmaven.test.failure.ignore=true'<br/>[2022-10-06T08:28:14.326Z] [INFO]  Executing pull request flow for https://github.com/kiegroup/optaplanner-quickstarts/pull/421 in bc<br/>[2022-10-06T08:28:14.326Z] (node:8164) Warning: Accessing non-existent property 'ClientError' of module exports inside circular dependency<br/>[2022-10-06T08:28:14.326Z] (Use `node --trace-warnings ...` to show where the warning was created)<br/>[2022-10-06T08:28:14.888Z] [INFO]  https://raw.githubusercontent.com/kiegroup/kogito-pipelines/main/.ci/pull-request-config.yaml exists, using it<br/>[2022-10-06T08:28:15.146Z] ::group::[PRE] Executing pre section for https://raw.githubusercontent.com/kiegroup/kogito-pipelines/main/.ci/pull-request-config.yaml<br/>[2022-10-06T08:28:15.146Z] [INFO]  Execute command 'export BUILD_MVN_OPTS=" -nsu -ntp -fae -e -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false dependency:tree -Dmaven.wagon.httpconnectionManager.ttlSeconds=120 -Dmaven.wagon.http.retryHandler.count=3"' in dir '/home/jenkins/workspace/KIE/optaplanner/main/pullrequest/optaplanner-quickstarts.tests.optaplanner-quickstarts'<br/>[2022-10-06T08:28:15.146Z] [INFO]  Treating export command since it's not possible to run it from the runner itself<br/>[2022-10-06T08:28:15.146Z] [INFO]  The variable BUILD_MVN_OPTS has been stored with ' -nsu -ntp -fae -e -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false dependency:tree -Dmaven.wagon.httpconnectionManager.ttlSeconds=120 -Dmaven.wagon.http.retryHandler.count=3' value<br/>[2022-10-06T08:28:15.146Z] ::notice title= [PRE] [Execution OK]::export BUILD_MVN_OPTS=" -nsu -ntp -fae -e -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false dependency:tree -Dmaven.wagon.httpconnectionManager.ttlSeconds=120 -Dmaven.wagon.http.retryHandler.count=3"<br/>[2022-10-06T08:28:15.146Z] [INFO]  Execute command 'echo "BUILD_MVN_OPTS= -nsu -ntp -fae -e -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false dependency:tree -Dmaven.wagon.httpconnectionManager.ttlSeconds=120 -Dmaven.wagon.http.retryHandler.count=3"' in dir '/home/jenkins/workspace/KIE/optaplanner/main/pullrequest/optaplanner-quickstarts.tests.optaplanner-quickstarts'<br/>[2022-10-06T08:28:15.146Z] [command]/usr/bin/echo BUILD_MVN_OPTS= -nsu -ntp -fae -e -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false dependency:tree -Dmaven.wagon.httpconnectionManager.ttlSeconds=120 -Dmaven.wagon.http.retryHandler.count=3<br/>[2022-10-06T08:28:15.146Z] BUILD_MVN_OPTS= -nsu -ntp -fae -e -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false dependency:tree -Dmaven.wagon.httpconnectionManager.ttlSeconds=120 -Dmaven.wagon.http.retryHandler.count=3<br/>[2022-10-06T08:28:15.147Z] ::notice title= [PRE] [Execution OK]::echo "BUILD_MVN_OPTS= -nsu -ntp -fae -e -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false dependency:tree -Dmaven.wagon.httpconnectionManager.ttlSeconds=120 -Dmaven.wagon.http.retryHandler.count=3"<br/>[2022-10-06T08:28:15.147Z] [INFO]  Execute command 'export BUILD_MVN_OPTS_CURRENT="-Dfull"' in dir '/home/jenkins/workspace/KIE/optaplanner/main/pullrequest/optaplanner-quickstarts.tests.optaplanner-quickstarts'<br/>[2022-10-06T08:28:15.147Z] [INFO]  Treating export command since it's not possible to run it from the runner itself<br/>[2022-10-06T08:28:15.147Z] [INFO]  The variable BUILD_MVN_OPTS_CURRENT has been stored with '-Dfull' value<br/>[2022-10-06T08:28:15.147Z] ::notice title= [PRE] [Execution OK]::export BUILD_MVN_OPTS_CURRENT="-Dfull"<br/>[2022-10-06T08:28:15.147Z] [INFO]  Execute command 'echo "BUILD_MVN_OPTS_CURRENT=-Dfull"' in dir '/home/jenkins/workspace/KIE/optaplanner/main/pullrequest/optaplanner-quickstarts.tests.optaplanner-quickstarts'<br/>[2022-10-06T08:28:15.147Z] [command]/usr/bin/echo BUILD_MVN_OPTS_CURRENT=-Dfull<br/>[2022-10-06T08:28:15.147Z] BUILD_MVN_OPTS_CURRENT=-Dfull<br/>[2022-10-06T08:28:15.147Z] ::notice title= [PRE] [Execution OK]::echo "BUILD_MVN_OPTS_CURRENT=-Dfull"<br/>[2022-10-06T08:28:15.147Z] ::endgroup::<br/>[2022-10-06T08:28:15.147Z] ::group::[Pull Request Flow] Execution Plan...<br/>[2022-10-06T08:28:15.147Z] [INFO]  ----------------------------------------------<br/>[2022-10-06T08:28:15.147Z] [INFO]  [2] projects will be executed<br/>[2022-10-06T08:28:15.147Z] <br/>[2022-10-06T08:28:15.147Z] [INFO]  [kiegroup/optaplanner]<br/>[2022-10-06T08:28:15.147Z] [INFO]  Level Type: [upstream].<br/>[2022-10-06T08:28:15.147Z] [INFO]  mvn clean install -Dquickly ${{ env.BUILD_MVN_OPTS }} ${{ env.BUILD_MVN_OPTS_UPSTREAM }} ${{ env.OPTAPLANNER_BUILD_MVN_OPTS_UPSTREAM }}<br/>[2022-10-06T08:28:15.147Z] [INFO]  [ [32m'docker system prune -f'[39m ]<br/>[2022-10-06T08:28:15.147Z] <br/>[2022-10-06T08:28:15.147Z] [INFO]  [kiegroup/optaplanner-quickstarts]<br/>[2022-10-06T08:28:15.147Z] [INFO]  Level Type: [current].<br/>[2022-10-06T08:28:15.147Z] [INFO]  mvn clean install -f optaplanner-quickstarts/pom.xml ${{ env.BUILD_MVN_OPTS }} ${{ env.BUILD_MVN_OPTS_CURRENT }} ${{ env.OPTAPLANNER_QUICKSTARTS_BUILD_MVN_OPTS }}<br/>[2022-10-06T08:28:15.147Z] [INFO]  [ [32m'docker system prune -f'[39m ]<br/>[2022-10-06T08:28:15.147Z] <br/>[2022-10-06T08:28:15.147Z] [INFO]  ----------------------------------------------<br/>[2022-10-06T08:28:15.147Z] ::endgroup::<br/>[2022-10-06T08:28:15.147Z] ::group::[Pull Request Flow] Checking out kiegroup/optaplanner-quickstarts and its dependencies<br/>[2022-10-06T08:28:15.147Z] [INFO]  Tree for project kiegroup/optaplanner-quickstarts. Dependencies: <br/>[2022-10-06T08:28:15.147Z] kiegroup/optaplanner,<br/>[2022-10-06T08:28:15.147Z] kiegroup/optaplanner-quickstarts<br/>[2022-10-06T08:28:15.147Z] [INFO]  [kiegroup/optaplanner] Checking out project<br/>[2022-10-06T08:28:15.711Z] [WARN]  project github.com/lampajr/optaplanner:BXMSPROD-1826_prod does not exist. It's not necessarily an error.<br/>[2022-10-06T08:28:15.711Z] [WARN]  project github.com/kiegroup/optaplanner:BXMSPROD-1826_prod does not exist. It's not necessarily an error.<br/>[2022-10-06T08:28:15.711Z] [WARN]  project github.com/kiegroup/optaplanner:stable does not exist. It's not necessarily an error.<br/>[2022-10-06T08:28:15.711Z] [ERROR]  [kiegroup/optaplanner] Trying to checking out kiegroup/optaplanner into 'bc/kiegroup_optaplanner'. It does not exist.<br/>[Pipeline] }<br/>[Pipeline] // withCredentials<br/>[Pipeline] }<br/>[2022-10-06T08:28:15.771Z] Deleting 1 temporary files<br/>[Pipeline] // configFileProvider<br/>[Pipeline] }<br/>[Pipeline] // script<br/>Post stage<br/>[Pipeline] junit<br/>[2022-10-06T08:28:15.855Z] Recording test results<br/>[2022-10-06T08:28:15.936Z] No test report files were found. Configuration error?<br/>Error when executing always post condition:<br/>Also:   hudson.remoting.Channel$CallSiteStackTrace: Remote call to rhos-01-kie-psi-rhel7-xlarge-167<br/>		at hudson.remoting.Channel.attachCallSiteStackTrace(Channel.java:1784)<br/>		at hudson.remoting.UserRequest$ExceptionResponse.retrieve(UserRequest.java:356)<br/>		at hudson.remoting.Channel.call(Channel.java:1000)<br/>		at hudson.FilePath.act(FilePath.java:1194)<br/>		at hudson.FilePath.act(FilePath.java:1183)<br/>		at hudson.tasks.junit.JUnitParser.parseResult(JUnitParser.java:118)<br/>		at hudson.tasks.junit.JUnitResultArchiver.parse(JUnitResultArchiver.java:159)<br/>		at hudson.tasks.junit.JUnitResultArchiver.parseAndSummarize(JUnitResultArchiver.java:253)<br/>		at hudson.tasks.junit.pipeline.JUnitResultsStepExecution.run(JUnitResultsStepExecution.java:63)<br/>		at hudson.tasks.junit.pipeline.JUnitResultsStepExecution.run(JUnitResultsStepExecution.java:29)<br/>		at org.jenkinsci.plugins.workflow.steps.SynchronousNonBlockingStepExecution.lambda$start$0(SynchronousNonBlockingStepExecution.java:47)<br/>		at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)<br/>		at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)<br/>		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)<br/>		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)<br/>		at java.base/java.lang.Thread.run(Thread.java:829)<br/>hudson.AbortException: No test report files were found. Configuration error?<br/>	at hudson.tasks.junit.JUnitParser$ParseResultCallable.invoke(JUnitParser.java:184)<br/>	at hudson.FilePath$FileCallableWrapper.call(FilePath.java:3502)<br/>	at hudson.remoting.UserRequest.perform(UserRequest.java:211)<br/>	at hudson.remoting.UserRequest.perform(UserRequest.java:54)<br/>	at hudson.remoting.Request$2.run(Request.java:376)<br/>	at hudson.remoting.InterceptingExecutorService.lambda$wrap$0(InterceptingExecutorService.java:78)<br/>	at java.util.concurrent.FutureTask.run(FutureTask.java:266)<br/>	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)<br/>	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)<br/>	at java.lang.Thread.run(Thread.java:750)<br/><br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>[2022-10-06T08:28:16.250Z] + rm -rf console.log<br/>[Pipeline] sh<br/>
</details>

**Pull request closed without merging**